client/components/LevelOne.jsx

```diff
@@ -2,12 +2,61 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const LevelOne = (props) => {
+  const titleStyle = {
+    fontSize: '20px',
+    fontWeight: '600',
+  };
+
+  const largeSelectable = {
+    marginTop: '10px',
+    width: '100%',
+    fontSize: '14px',
+    border: '1px solid #ccc',
+    height: '34px',
+    fontFamily: 'Helvetica',
+  };
+
+  const leftSmallSlectable = {
+    marginTop: '10px',
+    marginRight: '5%',
+    width: '47.5%',
+    fontSize: '14px',
+    border: '1px solid #ccc',
+    height: '34px',
+    marginBottom: '10px',
+  };
+
+  const rightSmallSelectable = {
+    marginTop: '10px',
+    width: '47.5%',
+    fontSize: '14px',
+    border: '1px solid #ccc',
+    height: '34px',
+    marginBottom: '10px',
+  };
+
+  const findTableButton = {
+    width: '100%',
+    color: 'white',
+    background: '#3cb52e',
+    borderColor: '#28781e',
+    boxShadow: '0 1px 1px rgba(0,0,0,0.3)',
+    borderRadius: '5px',
+    padding: '5px 8px',
+    fontSize: '12px',
+    fontWeight: '600',
+    lineHeight: '1.5em',
+    height: '34px',
+
+  };
+
+
   const {
     nextTwoWeeks, times, max, next, change, count, date, time,
   } = props;
 
   const guestCount = [];
-  for (let j = 1; j <= 12; j += 1) {
+  for (let j = 2; j <= 12; j += 1) {
```

Comment:
This is more of a suggestion. I don't know what 2 or 12 mean, maybe declare a variable and use them here. And people usually do j++ instead of j+=1. just minor thing.